src/Property/Property.java

```diff
@@ -0,0 +1,66 @@
+package Property;
+
+/**
+ * Created by darionmoore on 11/29/16.
+ */
+//  * Name: doesn't change
+//	* Price: doesn't change
+//	* Mortgage value: doesn't change
+//	* Color: doesn't change
+//	* Owned: can change
+
+    public class Property {
+
+    private String name;
+    private int price;
+    private int mortgageValue;
+    private String color;
+    private boolean owned;
```

Comment:
Not critical but I would side towards having an is prefix in front of boolean variables.  `isOwned` instead of just `owned`.